src/Container.tsx

```diff
@@ -16,12 +17,14 @@ const iOS = Platform.OS === "ios";
 
 export interface DialogContainerProps {
   blurComponentIOS?: ReactNode;
+  buttonStyle?: ViewStyle;
```

Comment:
A couple of questions:
- What do you feel this style should be used for?
- Why do you feel this should be added here instead of in the `Button` component?  

Personally, I would avoid exposing the button style prop at all (my preferred solution) to keep the API surface level to the minimum... or, if 100% needed (but let's chat about this), move it to the button component 👍

Reply:
@mmazzarolo I totally agree... when implementing I wasn't comfortable with the idea of messing with the touchable. At the same time, if I had that style prop, I would be able to achieve the vertical look as well.

I have no problems dropping this change.

Reply:
That would be great, thanks!